app/views/stats/index.html.slim

```diff
@@ -27,7 +27,7 @@ hr
             = "Action"
       tbody
         - @reports.each do |report|
-          - if FeatureSet.sars.disabled? && report.abbr == 'R103'
+          - if FeatureSet.sars.disabled? && report.abbr =~ /^R10\d$/   # the 100 series refer to SARS reports
```

Comment:
Would it be safer to use `/^R1\d\d$/` here? Otherwise we have an assumption that we'll always have `0` in the middle and the failure caused by this is likely to be easily overlooked.